src/SfdxProjectBuilder.groovy

```diff
@@ -859,8 +885,8 @@ class SfdxProjectBuilder implements Serializable {
 
   private void authenticateToDevHub() {
     _.echo('Authenticate to the Dev Hub ')
-    // _.echo(_.env.JWT_CRED_ID_DH)
-    _.withCredentials( [ _.file( credentialsId: _.env.JWT_CRED_ID_DH, variable: 'jwt_key_file') ] ) {
+    // _.echo(_.env.JWT_KEY_CRED_ID_DH)
+    _.withCredentials( [ _.file( credentialsId: _.env.JWT_KEY_CRED_ID_DH, variable: 'jwt_key_file') ] ) {
```

Comment:
We will need to remember to change the environment variable on this in the Jenkins config

Reply:
I was under the impression these were the variables defined in the reference file. I see now what you mean. Was twisted in my brain, I suppose.